packages/zoe/test/unitTests/contracts/test-automaticRefund.js

```diff
@@ -6,10 +6,11 @@ import { test } from 'tape-promise/tape';
 // eslint-disable-next-line import/no-extraneous-dependencies
 import bundleSource from '@agoric/bundle-source';
 
-import { makeZoe } from '../../../src/zoe';
+import { makeZoe } from '../../..';
```

Comment:
The lack of letters in the pathname weirds me out somewhat. If eslint will accept it, could we make this point to `../../../index` or `../../../index.js`? (it might not, in which case don't worry about it).

When we move away from `-r esm` to ESM proper, our `import` statements will need to point at fully-qualified files (no more `index.js` defaults), and I think that'll be slightly easier if our current imports point at files, rather than directories.

Reply:
eslint (via WebStorm) recommended this. I'd be happy if we configured eslint to recommend something else. 

I'd also be happy if we said it was good programming practice to ignore eslilnt recommendations, but that doesn't seem to be on the agenda.

Reply:
The suppression directive is also kinda ugly.

```
// noinspection ES6PreferShortImport
```